runtime/j9vm/javanextvmi.c

```diff
@@ -236,25 +236,25 @@ JVM_IsPreviewEnabled(void)
 }
 
 JNIEXPORT void JNICALL
-JVM_VirtualThreadMountBegin(JNIEnv *env)
+JVM_VirtualThreadMountBegin(JNIEnv *env, jobject vthread, jboolean first_mount)
```

Comment:
minor thing with the param naming: 

please change `first_mount` to `firstMount` and same for the others

Reply:
Sure, I'll change the names to match those used by related functions in `thread.cpp`.

Reply:
Updated.